lms/views/lti_launches.py

```diff
@@ -1,8 +1,12 @@
+from __future__ import unicode_literals
+
 import logging
 from datetime import datetime
 
+from pyramid.i18n import TranslationString as _  # noqa: N813
```

Comment:
What's `N813`? This doesn't seem to be necessary (`make lint` still passes without it)